salt/pillar/consul_pillar.py

```diff
@@ -203,11 +210,14 @@ def get_conn(opts, profile):
     else:
         conf = opts_merged
 
-    consul_host = conf.get('consul.host', '127.0.0.1')
```

Comment:
This PR doesn't introduce any backward incompatibilities. Removed lines are not required since these values are set as defaults inside `Consul`